src/core/foundations/src/typography/types.ts

```diff
@@ -49,12 +51,13 @@ export type Fs = (
 	{
 		lineHeight,
 		fontWeight,
-		italic,
+		fontStyle,
```

Comment:
Do we still need to explicitly include the legacy `italic` in here? Presumably not, since it's compiling okay! It looks weird that it's missing though 🤔

Reply:
Oh interesting. I didn't spot this because it's not throwing a type error. I don't think it matters what it's called because this is a type definition... you can call it `arg0` or `arg1` for all TS cares. I think it's just syntactic constraints that mean you have to give it a name 🤷‍♂️

I've added it back in anyway.